klayout/drc/rule_decks/mim_b.drc

```diff
@@ -0,0 +1,149 @@
+# frozen_string_literal: true
+
+################################################################################################
+# Copyright 2022 GlobalFoundries PDK Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################################
+
+if MIM_OPTION == 'B'
+  #================================================
+  #-------------MIM CAPACITOR OPTION B-------------
+  #================================================
+
+  logger.info('Starting MIM Capacitor Option B derivations')
+
+  topmin1_metal_fusetop = topmin1_metal.interacting(fusetop)
+  mimtm_virtual = fusetop.sized(1.06.um).and(topmin1_metal_fusetop)
+
+  # Rule MIMTM.1: Minimum MiM bottom plate spacing to the bottom plate metal
+  ## (whether adjacent MiM or routing metal) is 1.2µm.
+  logger.info('Executing rule MIMTM.1')
+  mimtm1_l1 = topmin1_metal.separation(mimtm_virtual, transparent, 1.2.um)
+  mimtm1_l1.output('MIMTM.1', "MIMTM.1 : Minimum MiM bottom plate spacing to the bottom plate metal
+                    (whether adjacent MiM or routing metal): 1.2µm")
+  mimtm1_l1.forget
+
+  # Rule MIMTM.2: Minimum MiM bottom plate overlap of Vian-1 layer.
+  ## [This is applicable for Vian-1 within 1.06um oversize of FuseTop layer
```

Comment:
I don't see the `1.06um` check in the rule, is that expected?

Reply:
@proppy The `mimtm_virtual` layer that used in this rule is implemented to satisfy this value. Refer to line 27

Reply:
oh sorry, I missed this! would it make sense to move it closer to the MIMTM.2 rule (unless it's also used elsewhere)? or maybe add a comment that explain which rules it is used for.

Reply:
@proppy  Yes, It's used for 3 rules. Comment added